raiden_contracts/tests/test_channel_cooperative_settle.py

```diff
@@ -173,25 +230,28 @@ def test_cooperative_settle_channel_0(
     )
 
 
-@pytest.mark.skip(reason="Delayed until another milestone")
 def test_cooperative_settle_channel_00(
     custom_token: Contract,
     token_network: Contract,
     create_channel_and_deposit: Callable,
     get_accounts: Callable,
-    create_cooperative_settle_signatures: Callable,
+    create_withdraw_signatures: Callable,
     cooperative_settle_state_tests: Callable,
 ) -> None:
     (A, B, C) = get_accounts(3)
     deposit_A = 0
     deposit_B = 0
     balance_A = 0
     balance_B = 0
+    expiration = 100
```

Comment:
You could add `expiration=100` as a default to `create_withdraw_signatures` to make the tests a bit more concise.

Reply:
Great idea!

Reply:
Mh, as it is also used in every contract call this doesn't help much. I put it into a constant instead.